docs/downstream/odh-install-crc.md

```diff
@@ -0,0 +1,83 @@
+# Installing ODH using ArgoCD
+
+## Preparation
+
+First make sure that you have the OpenDataHub operator available in your OpenShift.
+
+```bash
+oc get packagemanifests -n openshift-marketplace | grep opendatahub-operator
+```
+
+If it's not present, then you need to troubleshoot the marketplace: https://github.com/operator-framework/operator-marketplace/issues/344
+
+
+Then we need to create a projects named `odh-operator` and `odh-deployment`. (The name of the project is hard-coded in the `kustomize` files in https://github.com/anishasthana/my-odh-install and you can change is there using `kustomize`.)
```

Comment:
> Then we need to create a projects named 
project * 

> https://github.com/anishasthana/my-odh-install 

This is @anishasthana 's personal repo, are we okay with linking there? is there a more official location? If not, then maybe we bring these files in to this org?

Reply:
I vote we create a new repo within the operatre first umbrella @HumairAK

Reply:
@martinpovolny I just copied it to https://github.com/operate-first/odh